src/index.js

```diff
@@ -1,2 +1,108 @@
+/*
+ * This script assumes AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are exported as environment variables
+ */
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const http = require('http');
+const isProduction = process.env.NODE_ENV === 'production';
+const outputFile = 'blog-posts-data.json';
```

Comment:
let's call it _wordpress-data.json_ for now, so it's easy to see in S3 where the source of the data came from specifically.